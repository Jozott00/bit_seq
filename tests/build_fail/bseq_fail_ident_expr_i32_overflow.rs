use bit_seq::bseq;

fn main() {
    let var = 0;
    let _ = bseq!(var:32);
}