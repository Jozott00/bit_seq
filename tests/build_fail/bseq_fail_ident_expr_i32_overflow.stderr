error: literal out of range for `i32`
 --> tests/build_fail/bseq_fail_ident_expr_i32_overflow.rs:5:19
  |
5 |     let _ = bseq!(var:32);
  |                   ^^^
  |
  = note: the literal `var` does not fit into the type `i32` whose range is `-2147483648..=2147483647`
  = help: consider using the type `u32` instead
  = note: `#[deny(overflowing_literals)]` on by default
